src/ReadTrainer/PlayText.js

```diff
@@ -257,7 +262,10 @@ const PlayText = inject("store", "morsePlayer")(
       render() {
         if (this.state.hidden) {
           return (
-            <PlayHiddenCard onShow={this.onShow} onRepeat={this.playText} />
+            <PlayHiddenCard
+              onShow={this.onShow}
+              onRepeat={this.manualPlayText}
```

Comment:
Wouldn't just calling playText directly work?

Reply:
I forgot to clean that up after refactoring.